package.json

```diff
@@ -38,13 +38,13 @@
     "which": "^1.3.0"
   },
   "devDependencies": {
-    "eslint": "^5.0.0",
-    "eslint-config-semistandard": "^14.0.0",
-    "eslint-config-standard": "^13.0.1",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-promise": "^4.0.0",
-    "eslint-plugin-standard": "^4.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-semistandard": "^15.0.0",
+    "eslint-config-standard": "^14.1.0",
```

Comment:
That needs to be `^13.0.1` unless `eslint-config-semistandard` publishes an update before we merge this.

Reply:
👍 These dependencies will be drop soon and replaced with out new package anyways. It was just a step to get the PR in place and files linted.